gtests/net/mptcp/sockopts/mptcp_unsupported_sockopts_tcp_inq.pkt

```diff
@@ -0,0 +1,8 @@
+-tolerance_usecs=100000
+`../../common/defaults.sh`
+
+// test 1 : TCP_INQ is not supported by MPTCP sockets yet.
+// Follow https://github.com/multipath-tcp/mptcp_net-next/issues/224 for progress  
```

Comment:
sorry to be picky about that but please remove trailing whitespace and configure your editor to do that automatically. Most of us have this configuration. It means that if we need to edit this file, we will end up with "undesired" modifications.

(it is a shame Github doesn't clearly show them: I noticed them because you just modified this line with one more space at the end and I was wondering why you edited this line, it didn't seem modified.)